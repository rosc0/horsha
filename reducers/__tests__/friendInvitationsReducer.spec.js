import friendInvitationsReducer, {
  initialState as state,
} from '../friendInvitations'
// import types from '../../constants/actionTypes'

// import { horse_picture, horse_friendInvitations } from '../../../test/api'

describe('Comments Reducer', () => {
  it('should return initial state', () => {
    expect(friendInvitationsReducer()).toBe(state)
    expect(friendInvitationsReducer()).toMatchSnapshot()
  })
})
