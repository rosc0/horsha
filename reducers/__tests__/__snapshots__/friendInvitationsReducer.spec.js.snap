// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments Reducer should return initial state 1`] = `
Object {
  "cursor": null,
  "error": null,
  "fetched": false,
  "fetching": false,
  "ignoredInvitations": Array [],
  "ignoredRemaining": 0,
  "pendingInvitations": Array [],
  "pendingRemaining": 0,
}
`;
